BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -701,6 +705,8 @@ extension StackLayout {
         }
 
         func alignSegments(to alignment: AlignmentID.Type) -> [Segment] {
+            let crossMagnitudes = measureMagnitudes()
```

Comment:
Does this introduce a second call in some situations, potentially where before we would've only had one before?

Reply:
> Does this introduce a second call in some situations, potentially where before we would've only had one before?

I don't think so. The idea here is that we previously did it in every call to `_crossSegments`, and now it's skipped in some cases.

Reply:
👍